cross/ffmpeg-2.2.1/digests

```diff
@@ -0,0 +1,3 @@
+ffmpeg-2.2.1.tar.gz SHA1 cf455dd900cd7749f2c51c644054a9b657b8b589
```

Comment:
While this may be necessary, it doesn't belong in this PR. Can you move it to a separate one?